lms/djangoapps/learner_home/views.py

```diff
@@ -537,7 +537,7 @@ class CourseRecommendationApiView(APIView):
 
     def get(self, request):
         """ Retrieves course recommendations details of a user in a specified course. """
-        if not should_show_amplitude_recommendations():
+        if not should_show_amplitude_recommendations_on_learner_home_page():
```

Comment:
I am not sure if this is the intention of this waffle flag. They will remove the recommendations from our current dashboard and we don't want that.

Reply:
please check now